inference/core/managers/active_learning.py

```diff
@@ -109,11 +117,18 @@ async def infer_from_request(
         self, model_id: str, request: InferenceRequest, **kwargs
     ) -> InferenceResponse:
         active_learning_eligible = kwargs.get(ACTIVE_LEARNING_ELIGIBLE_PARAM, False)
+        active_learning_disabled_for_request = kwargs.get(
```

Comment:
Should this be default False to match the default value on the request entities?